documentation/docs/steps/containerExecuteOnKubernetes.md

```diff
@@ -0,0 +1,58 @@
+# containerExecuteInsidePod
+
+## Description
+
+Executes a closure inside a pod with multiple containers. The containers can be chosen as part of the closure.
+
+## Prerequisites 
+* The Jenkins should be running on kubernetes.
+* An environment variable `ON_K8S` should be created on Jenkins and initialized to `true`.
+* The step should be invoked only inside a [`podTemplate`](https://jenkins.io/doc/pipeline/steps/kubernetes/#podtemplate-define-a-podtemplate-to-use-in-the-kubernetes-plugin)
+ 
+## Parameters
+
+| parameter          | mandatory | default                           | possible values            |
+| -------------------|-----------|-----------------------------------|----------------------------|
+| `script`      | no        | empty `globalPipelineEnvironment`                                |                            |
+| `containerMap`      | yes        |                                | `['maven:3.5-jdk-7':'maven']`                           |
```

Comment:
I understood possible values, as a list of possible values so that only these values are allowed. In your case it is more like an example.